inference/src/routers/generate.py

```
@@ -41,9 +41,16 @@ async def _auto_install_deps(backend: str) -> None:
         if event.get("status") == "error":
-            raise HTTPException(status_code=500, detail=event.get("message", f"Failed to install {backend} dependencies"))
+            raise HTTPException(
+                status_code=500,
+                detail=event.get(
+                    "message", f"Failed to install {backend} dependencies"
+                ),
+            )
     importlib.invalidate_caches()
-    logger.info(f"Dependencies for '{backend}' installed successfully, retrying generation")
+    logger.info(
+        f"Dependencies for '{backend}' installed successfully, retrying generation"
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)